Xamarin.PropertyEditing/ViewModels/PanelViewModel.cs

```diff
@@ -341,6 +349,30 @@ protected override void OnClearProperties ()
 			ArrangedPropertiesChanged?.Invoke (this, EventArgs.Empty);
 		}
 
+		internal override bool GetIsLastVariant (PropertyViewModel viewModel)
+		{
+			if (viewModel == null)
+				throw new ArgumentNullException (nameof (viewModel));
+			if (!viewModel.IsVariant)
+				throw new ArgumentException ($"{nameof (viewModel)} is not a variant", nameof (viewModel));
+
+			string groupKey = GetGroup (viewModel);
+			PanelGroupViewModel group = this.arranged[groupKey];
+
+			var list = group.GetList (viewModel);
+
+			int index = list.IndexOf (viewModel);
+			if (index == -1)
+				throw new KeyNotFoundException ($"{nameof (viewModel)} was not found");
+
+			if (++index == list.Count)
+				return true;
+			if (list[index] is PropertyViewModel pvm) {
```

Comment:
that's just `return (++index == list.Count) || (list[index] is PropertyViewModel pvm && !Equals (viewModel.Property, pvm.Property));`

Reply:
I find this easier to read, no need to change.